CHANGES.md

```diff
@@ -11,6 +11,12 @@ Whenever possible, link the given PR with the feature/fix.
 
 [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Development
+
+### Added
+
+* New setting `SPIDERFEEDER_INPUT_FORMAT` to set the file format or fallback to the file extension. [PR#18](https://github.com/ejulio/spider-feeder/pull/18)
```

Comment:
fallback → fall back (when used as a verb, also affects the README)

Reply:
Thanks @Gallaecio D: